e2e/nomostest/registry_server.go

```diff
@@ -97,32 +97,70 @@ func setupRegistry(nt *NT) error {
 	return nil
 }
 
-// setupRegistryClient handles registry authentication.
-// This is necessary on each test because gcloud auth tokens expire after 1h and
-// some test suites take longer than 1h.
+// setupRegistryClient handles registry authentication and port-forwarding.
+//
+// Re-auth is necessary on each test because gcloud auth tokens expire after 1h
```

Comment:
This comment is no longer accurate with the usage of the credential helper

Reply:
Fixed.